Spark/metrics.csv

```diff
@@ -0,0 +1,207 @@
+spark.executor.total_input_bytes.median,gauge,,byte,,The median total number of input bytes in the application's executors,0,spark,exe in byt med
+spark.executor.total_input_bytes.95percentile,gauge,,byte,,The 95th percentile total number of input bytes in the application's executors,0,spark,exe in byt 95
+spark.executor.total_input_bytes.count,rate,,byte,,The number of times the total number of input bytes in the application's executors was sampled,0,spark,exe in byt cnt
+spark.executor.total_shuffle_read.max,gauge,,byte,,The max total number of bytes read during a shuffle in the application's executors,0,spark,exe tot shfl rd mx
+spark.executor.total_shuffle_read.avg,gauge,,byte,,The average total number of bytes read during a shuffle in the application's executors,0,spark,exe tot shfl rd avg
+spark.executor.total_shuffle_read.median,gauge,,byte,,The median total number of bytes read during a shuffle in the application's executors,0,spark,exe tot shfl rd med
+spark.executor.total_shuffle_read.95percentile,gauge,,byte,,The 95th percentile total number of bytes read during a shuffle in the application's executors,0,spark,exe tot shfl rd 95
+spark.executor.total_shuffle_read.count,rate,,byte,,The number of times the total number of bytes read during a shuffle in the application's executors was sampled,0,spark,exe tot shfl rd cnt
+spark.executor.total_shuffle_write.max,gauge,,byte,,The max total number of bytes written during a shuffle in the application's executors,0,spark,exe tot shfl wrt mx
+spark.executor.total_shuffle_write.avg,gauge,,byte,,The average total number of bytes written during a shuffle in the application's executors,0,spark,exe tot shfl wrt avg
+spark.executor.total_shuffle_write.median,gauge,,byte,,The median total number of bytes written during a shuffle in the application's executors,0,spark,exe tot shfl wrt med
+spark.executor.total_shuffle_write.95percentile,gauge,,byte,,The 95th percentile total number of bytes written during a shuffle in the application's executors,0,spark,exe tot shfl wrt 95
+spark.executor.total_shuffle_write.count,rate,,byte,,The number of times the total number of bytes written during a shuffle in the application's executors was sampled,0,spark,exe tot shfl wrt cnt
+spark.executor.max_memory.max,gauge,,byte,,The max maximum memory used in the application's executors,0,spark,exe max mem mx
+spark.executor.max_memory.avg,gauge,,byte,,The average maximum memory used in the application's executors,0,spark,exe max mem avg
+spark.executor.max_memory.median,gauge,,byte,,The median maximum memory used in the application's executors,0,spark,exe max mem med
+spark.executor.max_memory.95percentile,gauge,,byte,,The 95th percentile maximum memory used in the application's executors,0,spark,exe max mem 95
+spark.executor.max_memory.count,rate,,byte,,The number of times the maximum memory used in the application's executors was sampled,0,spark,exe max mem cnt

```

Comment:
`spark.executor.max_memory` is not the maximum used memory, it's the maximum amount of memory available for Spark to cache RDDs in-memory (called "storage memory") in versions <1.6.0. And since 1.6.0 that memory is used for both caching & executing, you can find details here: https://0x0fff.com/spark-memory-management/

Overall, I'd call it the maximum amount of memory available for caching RDD blocks.